controller/main.go

```diff
@@ -117,14 +117,17 @@ func (c *controller) SetPools(l log.Logger, pools map[string]*config.Pool) contr
 
 func main() {
 	var (
-		port            = flag.Int("port", 7472, "HTTP listening port for Prometheus metrics")
-		namespace       = flag.String("namespace", os.Getenv("METALLB_NAMESPACE"), "config / memberlist secret namespace")
-		mlSecret        = flag.String("ml-secret-name", os.Getenv("METALLB_ML_SECRET_NAME"), "name of the memberlist secret to create")
-		deployName      = flag.String("deployment", os.Getenv("METALLB_DEPLOYMENT"), "name of the MetalLB controller Deployment")
-		logLevel        = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", logging.Levels.String()))
-		disableEpSlices = flag.Bool("disable-epslices", false, "Disable the usage of EndpointSlices and default to Endpoints instead of relying on the autodiscovery mechanism")
-		enablePprof     = flag.Bool("enable-pprof", false, "Enable pprof profiling")
-		enableWebhook   = flag.Bool("enable-webhook", false, "Enable validation webhook")
+		port               = flag.Int("port", 7472, "HTTP listening port for Prometheus metrics")
+		namespace          = flag.String("namespace", os.Getenv("METALLB_NAMESPACE"), "config / memberlist secret namespace")
+		mlSecret           = flag.String("ml-secret-name", os.Getenv("METALLB_ML_SECRET_NAME"), "name of the memberlist secret to create")
+		deployName         = flag.String("deployment", os.Getenv("METALLB_DEPLOYMENT"), "name of the MetalLB controller Deployment")
+		logLevel           = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", logging.Levels.String()))
+		disableEpSlices    = flag.Bool("disable-epslices", false, "Disable the usage of EndpointSlices and default to Endpoints instead of relying on the autodiscovery mechanism")
+		enablePprof        = flag.Bool("enable-pprof", false, "Enable pprof profiling")
+		enableWebhook      = flag.Bool("enable-webhook", false, "Enable validation webhook")
+		enableCertRotation = flag.Bool("enable-cert-rotation", true, "enable automatic generation and rotation of webhook TLS certificates/keys")
```

Comment:
I'd change to disableCertRotation, it's more intuitive having the default value to false.